client/src/App.js

```diff
@@ -3,12 +3,15 @@ import { BrowserRouter as Router, Route } from "react-router-dom";
 import "./App.css";
 import HomePage from "./components/HomePage";
 import Hall from "./components/Hall";
+import Header from './components/Header/header';
 
 class App extends Component {
   render() {
     return (
       <Router>
         <div>
```

Comment:
Use ```Fragment ``` to wrap more than one component together, better than use ```div``` [React.Fragment](https://reactjs.org/docs/fragments.html)